tests/slow/src/test/scala/org/scalafmt/internal/CorrectnessTest.scala

```diff
@@ -0,0 +1,33 @@
+package org.scalafmt.internal
+
+import scala.meta._
+import org.scalafmt.Options
+
+import scala.meta.testkit.StructurallyEqual
+
+object CorrectnessTest extends PropertyTest("correctness") {
```

Comment:
"correctness" can be interpreted in many ways. Throwing away comments can be treated as "incorrect". "idempotent AST" leaves less room for interpretation, IMO. Either way, would be good to write a one line summary defining explicitly what property is being tested here.